src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -0,0 +1,352 @@
+});
+```
+
+You can pass many different options to tracing, but it comes with reasonable defaults out of the box.
+[Spans]({%- link _documentation/performance/performance-glossary.md -%}#span) are instrumented for the following operations within a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction):
+
+- A [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) for every page load
+- All XHR/fetch requests as spans
+
+**Using Tracing Integration for Manual Instrumentation**
+
+Tracing will create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) on page load by default; all [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) that are created will be attached to it. Also, tracing will finish the transaction after the default timeout of 500ms of inactivity. The page is considered inactive if there are no pending XHR/fetch requests. If you want to extend the transaction's lifetime beyond 500ms, you can do so by adding more spans to the transaction. The following is an example of how you could profile a React component:
+
+```javascript
+// This line starts an activity (and creates a span).
+// As long as you don't pop the activity, the transaction will not be finished and therefore not sent to Sentry.
+// If you do not want to create a span out of an activity, just don't provide the second arg.
+const activity = ApmIntegrations.Tracing.pushActivity(displayName, {
+        data: {},
+        op: 'react',
+        description: `<${displayName}>`,
+});
+
+// Sometime later ...
+// When we pop the activity, the Integration will finish the span and after the timeout finish the transaction and send it to Sentry
+Integrations.ApmIntegrations.popActivity(activity);
+```
+
+**Manual Transactions**
+
+To manually instrument certain regions of your code, you can create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) to capture them.
+
+The following example creates a transaction for a scope that contains an expensive operation (for example, `process_item`), and sends the result to Sentry:
+
+```javascript
+const transaction = Sentry.getCurrentHub().startSpan({ op: "task",  transaction: item.getTransaction() })
+  // processItem may create more spans internally (see next example)
+  processItem(item).then(() => {
+    transaction.finish()
+  })
+```
+
+**Adding Additional Spans to the Transaction**
+
+The next example contains the implementation of the hypothetical `processItem ` function called from the code snippet in the previous section. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
```

Comment:
```suggestion
The next example contains the implementation of the hypothetical `processItem ` function called from the code snippet in the previous section. Our SDK can determine if there is a currently open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
```